src/services/reducers.js

```diff
@@ -27,6 +27,13 @@ const top = function(state = initialState.top, action) {
       return state;
   }
 };
+const postPhotoInstances = function(photos) {
+  const data = window.localStorage.hasOwnProperty('yori-photo') 
```

Comment:
```js
window.localStorage['yori-photo']
```
should be enough I think. Usually, you won't have `yori-photo` in your prototype chain.